### DISCUSSION TITLE
Possible to increase container count more then 250 on Concourse V7.5

### DISCUSSION THREAD

**Discussion by prime89-hash** [concourse/concourse#7669] [Help & Support]
Hi Team, 

Is it possible to increase the container limit more then 250 per node on concourse v7.5 ?

**Comment by taylorsilva** [comment]
Yes, it's always been possible. You'll need to do two things.

1. Increase the limit by setting `CONCOURSE_GARDEN_MAX_CONTAINERS` or `CONCOURSE_CONTAINERD_MAX_CONTAINERS` depending on the runtime you use. Garden is the default runtime and the one you're most likely using.
2. Ensure the network pool is large enough to also support more containers. You can set this with `CONCOURSE_GARDEN_NETWORK_POOL` or `CONCOURSE_CONTAINERD_NETWORK_POOL`. It defaults to `10.80.0.0/16` which gives you 255 IP's, one for each container. This is where the max of 250 containers comes from. Changing this to ~`10.80.0.0/8` may work, but I'm not a networking expert 😛~ `10.254.0.0/16` will give you a limit of ~65k IP's and thus containers.

**Reply by crsimmons** [reply]
One of my colleagues wrote a blog post last year on this with some tests for how high you can push the limit. https://www.engineerbetter.com/blog/increasing-concourse-container-limits/

**Comment by prime89-hash** [comment]
Thank @crsimmons and @taylorsilva , for sharing this will check and try to use on our concourse configuration.
